master/internal/resourcemanagers/priority.go

```diff
@@ -185,7 +185,8 @@ func trySchedulingTaskViaPreemption(
 	log.Debugf("trying to schedule task %s by preempting other tasks", allocationRequest.Name)
 
 	for priority := model.MaxUserSchedulingPriority; priority > allocationPriority; priority-- {
-		for _, preemptionCandidate := range priorityToScheduledTaskMap[priority] {
+		for i := len(priorityToScheduledTaskMap[priority]) - 1; i >= 0; i-- {
```

Comment:
nit golang, why no slice.Reverse...

Reply:
sort.Reverse? I didn't know about it. and that seems to not modify the array in place which we can't have.

Reply:
Oh I mean nit the go language lol